From 2325315be2373006c8afae5d77e9c447638aab87 Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Thu, 17 Jan 2019 16:43:25 +0100
Subject: [PATCH] MIPS: gardena-smart-gateway_defconfig: Enable INPUT_EVENT and
 KEYBOARD_GPIO

By enabling INPUT_EVENT and KEYBOARD_GPIO the reset GPIO input pin can
be accessed from userspace, e.g. via /dev/input/eventX.

Signed-off-by: Stefan Roese <sr@denx.de>
---
 arch/mips/configs/gardena-smart-gateway_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/mips/configs/gardena-smart-gateway_defconfig b/arch/mips/configs/gardena-smart-gateway_defconfig
index 8c2c76ce838a..21bdb38aedb5 100644
--- a/arch/mips/configs/gardena-smart-gateway_defconfig
+++ b/arch/mips/configs/gardena-smart-gateway_defconfig
@@ -133,9 +133,10 @@ CONFIG_NET_VENDOR_MEDIATEK=y
 CONFIG_PHYLIB=y
 CONFIG_SWCONFIG=y
 # CONFIG_WLAN is not set
-# CONFIG_INPUT_KEYBOARD is not set
+CONFIG_INPUT_EVDEV=y
+CONFIG_INPUT_EVBUG=y
+CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
-# CONFIG_SERIO is not set
 CONFIG_DEVKMEM=y
 CONFIG_SERIAL_8250=y
 CONFIG_SERIAL_8250_CONSOLE=y
